metaci/testresults/importer.py

```diff
@@ -158,6 +163,26 @@ def import_robot_test_results(build_flow, path):
         )
         testresult.save()
 
+        if result['screenshots']:
+            dirname = os.path.dirname(path)
+            for screenshot in result['screenshots']:
+                screenshot_path = screenshot
+                if dirname:
```

Comment:
When would this not be true?

Reply:
If the robot output files are located in the build dir rather than in app root, the dirname would be empty.  In that case we just pass the name of the file.  The dirname is only passed if the files are found in the app root dir, otherwise `screenshot` is just the file name with no path before it.  The need for this would go away if we can figure out why robot started writing these files into the app root instead of the temp build dir.

Reply:
Got it, dirname ends up empty if `path` is just a filename with no directory--that's the case I was overlooking. I did look a bit to see why robot is writing to the app root, but it's not immediately obvious.